atc/api/jobserver/badge.go

```diff
@@ -95,6 +98,15 @@ type badgeTemplateConfig struct {
 	StatusTextWidth string
 	Status          string
 	FillColor       string
+	Title           string
+}
+
+func enrichFromQuery(params url.Values, original *Badge) Badge {
```

Comment:
This code seems a little odd. I think we can just change `BadgeForBuild` to not return a pointer and then inline the `.title` attribute assignment, since then it would only be modifying a copy.

Reply:
I agree with you, this kind of stuff are a bit odd with Go.

Actually, what you are saying was my very first though and implementation. 
I changed it back because I didn't want to do such a big change, but if you're ok with that I'm going to change it back :D